tests/keyword_arguments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,28 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`optional.rb 1`] = `
```

Comment:
Github is becoming confused and treating the snapshot as a new file as it as a lot more tests now 🤷‍♂️